idaes/models_extra/power_generation/properties/tests/test_sofc_surrogates.py

```diff
@@ -53,6 +53,7 @@ def build_rom():
 
 
 @pytest.mark.unit
+@pytest.mark.xfail
```

Comment:
Could you add an inline comment to explain why this is `xfailed`. I know this is mentioned in the deprecation warning in the code, but having comments here will help reinforce that this issue needs to be fixed.

Reply:
I'll add the comments now. @AlexNoring is aware of the SOFC surrogate issue and will continue the discussion here: https://github.com/IDAES/idaes-pse/issues/1431